libtrip-notification/src/main/assets/open_source_licenses.html

```diff
@@ -0,0 +1,253 @@
+<html>
```

Comment:
At the end of generation script, after `LICENSE.md` is filled up, could you remove those files so that we don't have to commit them to the repo?